src/fu-util.c

```diff
@@ -2112,6 +2169,13 @@ main (int argc, char *argv[])
 		return EXIT_FAILURE;
 	}
 
+	/* impossible */
+	if (priv->assume_yes && priv->assume_no) {
+		/* TRANSLATORS: the user didn't read the man page */
+		g_print ("%s\n", _("Incompatible arguments specified"));
```

Comment:
Someone might have loaded up a ton of arguments, I think it's better to be explicit that `--assume-yes` and `--assume-no `are the ones causing problems.

Reply:
I think I'm confusing two things as one issue. `--assume-yes`/`-y` is a common idiom in CLI scripts, and I think makes sense. What I was trying to achieve with `--assume-no` is really "skip these actions" which I think is better served explicitly, rather than just trying to use a heuristic for "no" everywhere. See updated commit.